GeosUtil/timers_mod.F90

```diff
@@ -977,11 +987,12 @@ END FUNCTION Timer_TheTime
 !\\
 ! !INTERFACE:
 !
-  SUBROUTINE Timer_TimePrint( SlotNumber )
+  SUBROUTINE Timer_TimePrint( SlotNumber, Lun_JSON )
```

Comment:
Are there conventions in fortran about using different cases for variables? I know it is case insensitive, but I see `lun_json`, `Lun_JSON`, and `lun_JSON`.  I am still not used to it 🙃

Reply:
Good point @laestrada and @msulprizio.  I'll fix that.

Reply:
See commit 17583e387ddfeed56d17724912fb9be3a9e53993.  I now use camel case for the relevant JSON variables as this matches the convention used in `timers_mod.F90`.